tasks/create_app_database_vault.yml

```diff
@@ -14,6 +14,6 @@
 
 - name: Append database to databases vault
   set_fact:
-    databases: "{{ databases | default({}) | merge_with_database(item, app.name, customer, env_type) }}"
+    databases: "{{ databases | default({}) | merge_with_database(item, app.name, customer, env_types | json_query(\"[?name=='\" ~ env_type ~ \"'] | [0]\")) }}"
```

Comment:
I would not mind a comment explaining what the json query is selecting in plain english if its possible. I wanted to propose something but it looked like : selected the env_type from the env_types that matches the env_type :-(
A sign that the name of the `env_types` list is not optimal?

Reply:
Would you prefer `environments`? And the related comment:

> Select the environment matching the current `env_type`